components/PaginationLinks.js

```diff
@@ -9,6 +9,10 @@ const PaginationLinks = ({ offset, username }) => {
   const newerLink = '/' + username + (newerOffset ? '?offset=' + newerOffset : '');
   const olderLink = '/' + username + '?offset=' + olderOffset;
 
+  if (hideOnFirst && offset <= 0) {
```

Comment:
Hmm... Did you test this condition? I believe normally on the first page `offset` will be `undefined`, thus `offset <= 0` will be `false`, therefore PaginationLinks will _not_ be hidden. Unless I'm missing something.

Reply:
Alternatively, you can use `normalOffset` instead of `offset`, then your condition should work fine.